spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -39,6 +39,23 @@
     end
   end
 
+  context 'a belongs_to association with a presence validation and allow_nil' do
+    it 'allows a nil value' do
+      if ::ActiveModel::VERSION::STRING.to_f >= 4.0
```

Comment:
You don't need this check here -- we dropped support for Rails 3.x and now support 4+.